Source/Urho3D/Graphics/DebugRenderer.cpp

```diff
@@ -505,7 +541,7 @@ void DebugRenderer::Render()
     graphics->SetBlendMode(lineAntiAlias_ ? BLEND_ALPHA : BLEND_REPLACE);
     graphics->SetColorWrite(true);
     graphics->SetCullMode(CULL_NONE);
-    graphics->SetDepthWrite(true);

```

Comment:
As far as I understand, after it is rendered another viewport or UI. And every set own settings